content/metadata.yaml

```diff
@@ -524,12 +524,12 @@ author_info:
   #   twitter: null
   #   affiliations:
   #   - Boise State University
-  # - name: Trang Le
-  #   orcid: null
-  #   github: null
-  #   twitter: null
-  #   affiliations:
-  #   - University of Pennsylvania
+- name: Trang Le
```

Comment:
@trang1618 please try adding 2 spaces to the beginning of each of these lines so that they all have another level of indentation.  That should fix the build error.

Reply:
Thank you @agitter. I added the 2 spaces [here](https://github.com/dhimmel/psb-manuscript/pull/1/commits/c85fa7ab14c4e0c641e9abaf6fdfd26724123753).